tuna/import_configs.py

```diff
@@ -42,9 +42,8 @@
 
 def parse_args():
   """Parsing arguments"""
-  parser = setup_arg_parser(
-      'Import driver commands and perf db entries ',
-      [TunaArgs.ARCH, TunaArgs.NUM_CU, TunaArgs.VERSION, TunaArgs.CONFIG_TYPE])
+  parser = setup_arg_parser('Import driver commands and perf db entries ',
```

Comment:
With the idea that at some point this will become MIOpen code only, can we update this comment with: Import MIOpen Driver commands and MIOpen performance DB entries.